bench-vortex/src/public_bi.rs

```diff
@@ -406,31 +403,28 @@ impl Dataset for PBIBenchmark {
         &self.name
     }
 
-    // TODO(osatici): compress benchmarks use this, but this relies on all
-    //                tables in a benchmark to have the same schema.
-    //                That is not the case for some PBI datasets.
     async fn to_vortex_array(&self) -> ArrayRef {
         let dataset = self.dataset().expect("failed to parse tables");
         dataset.write_as_vortex().await;
-
-        let arrays = stream::iter(dataset.list_files(FileType::Vortex))
-            .map(|f| async move {
-                VortexOpenOptions::file()
-                    .open(f)
-                    .await?
-                    .scan()?
-                    // TODO(ngates): why do we spawn this on the tokio executor?
-                    .with_tokio_executor(Handle::current())
-                    .into_array_stream()?
-                    .read_all()
-                    .await
-            })
-            .buffered(10)
-            .try_collect::<Vec<_>>()
-            .await
-            .unwrap();
-
-        ChunkedArray::from_iter(arrays).into_array()
+        // reading only the first table, each table in a PBI benchmark
+        // has its own schema.
```

Comment:
this should only affect the cms provider numbers, that was the only multi file public bi dataset that we were compressing. That luckily had the same schema for all its files but that is not true for all public bi datasets